src/lib/tabs/tab-nav-bar/tab-nav-bar.spec.ts

```diff
@@ -125,6 +125,23 @@ describe('MatTabNavBar', () => {
         .toBe(true, 'Expected element to no longer be keyboard focusable if disabled.');
     });
 
+    it('should prevent default action for clicks if links are disabled', () => {
+      const tabLinkElement = fixture.debugElement.query(By.css('a')).nativeElement;
+
+      const mouseEvent = new MouseEvent('click');
```

Comment:
Can you use `createMouseEvent('click')` instead? This uses a more compatible (e.g. IE) logic for initializing the event.